### PR TITLE
Add missing Theater location translation

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Fixed

- Add missing Theater location translation

---

We have the translation for Theater as an event but not as a type of location from what I understand, so added it with the other ID.

Ticket: https://jira.uitdatabank.be/browse/III-5494

### CODE REVIEW

**Review by simon-debruijn — APPROVED** [review]

File: src/i18n/de.json [hunk 1]
```diff
@@ -585,6 +585,7 @@
     "0.53.0.0.0": "Freizeitbereich oder -zentrum",
     "0.54.0.0.0": "Tanzvorführung",
     "0.55.0.0.0": "Theatervorstellung",
+    "8.70.0.0.0": "Theatervorstellung",
```
Comment (corneelwille) [inline src/i18n/de.json:588]:
```suggestion
    "8.70.0.0.0": "Theater",
```

File: src/i18n/nl.json [hunk 2]
```diff
@@ -584,6 +584,7 @@
     "0.53.0.0.0": "Recreatiedomein of centrum",
     "0.54.0.0.0": "Dansvoorstelling",
     "0.55.0.0.0": "Theatervoorstelling",
+    "8.70.0.0.0": "Theatervoorstelling",
```
Comment (corneelwille) [inline src/i18n/nl.json:587]:
```suggestion
    "8.70.0.0.0": "Theater",
```

**Comment by corneelwille** [comment]
@Anahkiasen I did two suggestions because the label / name that is used is incorrect
* `8.70.0.0.0` is the identifier for `Theatre` (the **location** / venue where participants can go to a theatre performance) not for the (theatre performance itself). 
* The identifier for the theatre performance (the **event**) was already in this file (`0.55.0.0.0`) with its corresponding label

**Comment by Anahkiasen** [comment]
@corneelwille Not sure I understand your message, do you mean the change in the PR is incorrect then? 🤔 Not sure I get what you mean by "two suggestions"

**Comment by corneelwille** [comment]
> @corneelwille Not sure I understand your message, do you mean the change in the PR is incorrect then? 🤔 Not sure I get what you mean by "two suggestions"

@Anahkiasen yes the PR is incorrect. The ticket was about places and not about events: https://jira.uitdatabank.be/browse/III-5494
The issue is that the correct label for `8.70.0.0.0` is `Theater` (dutch for the venue / **place** where you can go to). But in this PR `Theatervoorstelling` is used (dutch for the *event* you can attend / theatre performance).

**Comment by corneelwille** [comment]
Extra context: see our Taxonomy API: https://taxonomy.uitdatabank.be/terms

**events**
```
"id":"0.55.0.0.0",
"domain":"eventtype",
{
"name":{
   "nl":"Theatervoorstelling",
   "fr":"Théâtre",
   "de":"Theater",
   "en":"Theatre"
},
"scope":[
   "events"
],
```

**places**
```
{
   "id":"8.70.0.0.0",
   "domain":"eventtype",
   "name":{
      "nl":"Theater",
      "fr":"Théâtre",
      "de":"Theater",
      "en":"Theatre"
   },
   "scope":[
      "places"
   ]
}
```

**Review by brampauwelyn — APPROVED** [review]

**Pull request merged**